product_docs/docs/pgd/6/get-started/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Get started with PGD
+navTitle: Get started
+description: "Get started with EDB Postgres Distributed, installing and configuring the software, and creating your first cluster."
+navigation:
+- essential-standard
+- essential-near-far
+- first-cluster
+- expanded-examples
+---
+
+To begin using any edition of EDB Postgres Distributed, we recommend you first try our local installation and configuration guide. This guide will help you set up a Docker-compose cluster on your local machine, which is ideal for testing and learning purposes.
```

Comment:
in product_docs/docs/pgd/6/get-started/first-cluster/index.mdx 
there is a `Docker Compose` 
while here is `Docker-compose`

Perhaps unify to `Docker Compose`?